src/xclim/sdba/__init__.py

```diff
@@ -2,15 +2,30 @@
 ==============
 SDBA submodule
 ==============
+
+.. warning::
+
+    The SDBA submodule is in the process of being split from `xclim` in order to facilitate effective maintenance
+    of the SDBA utilities as well as speed up imports in `xclim`. The `xclim.sdba` API will be deprecated in the future.
+    For more information, please visit https://xsdba.readthedocs.io/en/latest/.
 """
 
 from __future__ import annotations
 
+import warnings
+
 from . import adjustment, detrending, measures, processing, properties, utils
 from .adjustment import *
 from .base import Grouper
 from .processing import stack_variables, unstack_variables
 
+warnings.warn(
+    "The SDBA module is in the process of being split from 'xclim' in order to facilitate effective maintenance "
+    "of the SDBA utilities as well as speed up imports in `xclim`. The `xclim.sdba` API will be deprecated in the future."
+    "For more information, please visit https://xsdba.readthedocs.io/en/latest/.",
+    FutureWarning,
+)
```

Comment:
```suggestion
warnings.warn(
    "The SDBA module is in the process of being split from 'xclim' in order to facilitate development and "
    " effective maintenance of the SDBA utilities. The `xclim.sdba` API will be deprecated in the future."
    "For more information, please visit https://xsdba.readthedocs.io/en/latest/.",
    FutureWarning,
)
```
I don't think the removal of sdba will speed up any non-sdba parts of xclim (except `xclim.testing`, but that's not user API). And I suggest mentioning that the goal is also to make development easier.